### PR TITLE
Doc CLI option for license key

### PR DESCRIPTION
**Pull request by WanYixian**
## Description
Doc CLI option for license key

## Related code PR 
https://github.com/risingwavelabs/risingwave/pull/18768

## Related doc issue
Resolve https://github.com/risingwavelabs/risingwave-docs-new/issues/52

### CODE REVIEW

File: get-started/rw-premium-edition-intro.mdx [hunk 1]
```diff
@@ -58,6 +58,13 @@ SELECT rw_test_paid_tier();
 
 A result of `t` means the key is valid; an error message indicates an invalid key.
 
+### CLI option for license key
```
Comment (BugenZhao) [inline get-started/rw-premium-edition-intro.mdx:61]: I think we'd better merge this section into `### Set license key` as an alternative (and automated) method of setting the license key.

Comment (WanYixian) [reply get-started/rw-premium-edition-intro.mdx]: Updated, please take another look

File: get-started/rw-premium-edition-intro.mdx [hunk 2]
```diff
@@ -48,6 +52,15 @@ To set your license key:
 ALTER SYSTEM SET license_key TO '...';
 ```
 
+#### Automated method
+
+To set your license key automatically:
+
+1. Use the `--license-key-file` CLI option for the meta node to monitor and reload the license key from a specified file. This streamlines license key rotation in Cloud environments.
```
Comment (BugenZhao) [inline get-started/rw-premium-edition-intro.mdx:59]: `--license-key-file` was later renamed to `--license-key-path` to be consistent with the env var key in https://github.com/risingwavelabs/risingwave/pull/18838. Would you help to update it? Same for other occurrences.

**Review by BugenZhao — APPROVED** [review]
LGTM

**Pull request merged**